Gemfile.lock

```diff
@@ -6,14 +6,14 @@ GEM
     nokogiri (1.6.8.1)
       mini_portile2 (~> 2.1.0)
     puma (3.6.0)
-    rack (1.6.5)
+    rack (1.5.2)
```

Comment:
Yes it was a docker fs cache problem. Im reverting the changes on versions.
Thanks!